README.md

```diff
@@ -470,6 +471,25 @@ client.get(options, function(err, res, socket, head) {
 });
 ```
 
+### More APIs
+
+The following methods are shared by all versions of the client:
+
+#### inflightRequests()
+
+Returns the number of inflight requests the client is currently handling. An
+inflight request is a request that is in any of the following states:
+
+* connection establishment (dns resolution, socket connection, etc.)
```

Comment:
Should we add an item "* After the verb function is called but before any I/O (dns resolution, socket connection, etc.) is performed."? It's mentioned below but here it seems this list is meant to be exhaustive, so it seems it'd be misleading to not mention this state as part of the included states.